File: Chapter 02/C Programs/2_14.c
#include<stdio.h>
void main()
{
int x,y;
char op;
printf("Enter two numbers: ");
scanf("%d%d",&x, &y);
fflush(stdin);
printf("Enter operator: a for addition, s for subtraction, m for multiplication and d for division: ");
scanf("%c", &op);
if (op== 'a')
printf("%d",x+y);
else if (op== 's')
printf("%d", x-y);
else if (op== 'm')
printf("%d", x*y);
else if (op== 'd')
printf("%f", x/(float)y);
else
printf( "Entered invalid operator");
}

File: Chapter 02/Python programs/2_14.py
x=int(input("Enter first number "))
y= int(input("Enter second number "))
op=input("Enter operator: a for addition, s for subtraction, m for multiplication and d for division ")
if op== 'a':
    print( x+y)
elif op== 's':
    print( x-y)
elif op== 'm':
    print( x*y)
elif op== 'd':
    print( x/y)
else:
    print("Entered invalid operator")
